dapr/proto/runtime/v1/dapr.proto

```diff
@@ -319,6 +319,7 @@ message RegisterActorTimerRequest {
   string period = 5;
   string callback = 6;
   bytes  data = 7;
+  uint32 repeats = 8;
```

Comment:
Can we change the approach to be time based? I think it will avoid having people to do the math for how many times they want it triggered given the deadline. Since the rest of the API is time based (due_time and period), I think that keeping this one time based will bring consistency.

/cc @yaron2  thoughts?

Reply:
Ideally we should support both - give the user choice to select either duration or number of repetitions.
I suggest adding another optional field `string duration = 9;` to specify timer duration. We will throw an error if both `repeats` and `duration` are set.

Reply:
I like that too.